plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/sessionconfig/CodeScanSessionConfig.kt

```diff
@@ -159,6 +164,23 @@ class CodeScanSessionConfig(
         }
     }.toFile()
 
+    private fun zipFile(filePath: Path): File = createTemporaryZipFile {
+        try {
+            val relativePath = filePath.relativeTo(projectRoot.toNioPath())
+            val virtualFile = VirtualFileManager.getInstance().findFileByNioPath(filePath)
```

Comment:
these two lines are error prone in case of remote development cases. i'll make the changes for these two once i complete testing for remote cases.

Reply:
Discussed offline, this will be addressed in a separate PR